<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LatituteE7340" failures="2" tests="14" name="Test" time="86.647" errors="0" timestamp="2023-05-01T11:43:15 IST">
  <testcase classname="testEndPoints.AdequateEndpointTests" name="TestUserRegister" time="5.975"/>
  <testcase classname="testEndPoints.AdequateEndpointTests" name="TestValidUserLogin" time="0.48"/>
  <testcase classname="testEndPoints.AdequateEndpointTests" name="TestCreateUser" time="0.442"/>
  <testcase classname="testEndPoints.AdequateEndpointTests" name="TestgetUser" time="0.382"/>
  <testcase classname="testEndPoints.AdequateEndpointTests" name="TestInValidUserLogin" time="0.265"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testLoginUser" time="61.065"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testRegisterUser" time="0.606"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testCreateProducts" time="10.688">
    <failure type="org.awaitility.core.ConditionTimeoutException" message="Condition with lambda expression in testEndPoints.MobileStoreEndPointTests that uses io.restassured.response.Response was not fulfilled within 10 seconds.">
      <![CDATA[org.awaitility.core.ConditionTimeoutException: Condition with lambda expression in testEndPoints.MobileStoreEndPointTests that uses io.restassured.response.Response was not fulfilled within 10 seconds.
at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:136)
at org.awaitility.core.CallableCondition.await(CallableCondition.java:79)
at org.awaitility.core.CallableCondition.await(CallableCondition.java:27)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:840)
at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:802)
at testEndPoints.MobileStoreEndPointTests.testCreateProducts(MobileStoreEndPointTests.java:107)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testCreateProducts -->
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testCreateProducts" time="6.099"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testCreateProducts" time="5.778"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testGetProduct" time="0.465"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testUpdateProduct" time="0.377"/>
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testDeleteProduct" time="0.48">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;200&amp;gt; but was &amp;lt;502&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <502>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$2.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
at io.restassured.specification.ResponseSpecification$statusCode$1.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at testEndPoints.MobileStoreEndPointTests.testDeleteProduct(MobileStoreEndPointTests.java:190)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteProduct -->
  <testcase classname="testEndPoints.MobileStoreEndPointTests" name="testGetProducts" time="0.0">
    <skipped/>
  </testcase> <!-- testGetProducts -->
</testsuite> <!-- Test -->
